modules/@apostrophecms/doc-type/ui/apos/components/AposDocVersions.vue

```diff
@@ -20,7 +20,7 @@
       <AposModalBody>
         <template #bodyMain>
           <AposTree
-            :rows="rows" :headers="headers"
```

Comment:
Renames rows to items to be consistent, and since media manager doesn't have visual rows.

Reply:
We could use "docs" across the board instead. That could make more sense.